takwimu/templates/profile/profile_detail_takwimu.html

```diff
@@ -4,7 +4,14 @@
 {% block profile_header_tail %}
 {% endblock %}
 
-
+{% block head_css_extra %}
```

Comment:
QQ: Why are these here and not in `takwimu.scss` @karimkawambwa?

Reply:
At the moment fix, I guess i can move it there.

Reply:
Lets see if @nerdlet  notices that the fix works. Hopefully she has the time.